assignments/1. counter/index.html

```diff
@@ -1,10 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Basic Counter App</title>
-  </head>
-  <body></body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<title>Basic Counter App</title>
+		<style>
+			.container {
+				margin: 0 auto;
+				text-align: center;
+				font-family: "Helvetica", sans-serif;
+			}
+			h1 {
+				font-size: 42px;
+				text-shadow: 2px 2px 0px rgba(0, 0, 0, 0.2);
+			}
+			p {
+				font-size: 60px;
+				font-family: monospace;
+			}
+			.btn {
+				padding: 20px 40px;
+				margin-right: 30px;
+				font-size: 22px;
+				border-radius: 5px;
+				box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2),
+					0px 2px 2px 0px rgba(0, 0, 0, 0.14),
+					0px 1px 5px 0px rgba(0, 0, 0, 0.12);
+				outline: none;
+				box-sizing: border-box;
+			}
+			#increment {
+				background-color: #09af00;
+			}
+			#decrement {
+				background-color: #e53935;
+			}
+			#reset {
+				background-color: #2979ff;
+			}
+		</style>
+	</head>
+	<body>
+		<div class="container">
+			<h1>Basic Counter App</h1>
+			<p class="countValue">0</p>
+			<button class="btn" id="increment">Increment</button>
+			<button class="btn" id="decrement">Decrement</button>
+			<button class="btn" id="reset">Reset</button>
+		</div>
+		<script>
+			document
+				.querySelector("#increment")
+				.addEventListener("click", increment);
+			document
+				.querySelector("#decrement")
+				.addEventListener("click", decrement);
+			document.querySelector("#reset").addEventListener("click", reset);
+			function increment() {
+				var count = document.querySelector(".countValue");
+				count.textContent = Number(count.textContent) + 1;
+			}
+			function decrement() {
+				var count = document.querySelector(".countValue");
+				count.textContent = Number(count.textContent) - 1;
+			}
+			function reset() {
+				var count = document.querySelector(".countValue");
+				count.textContent = 0;
+			}
```

Comment:
Try to separate things like

- Selecting element
- Logic
- AddEventListeners

When you separate code it becomes easier to maintain. It will much cleaner if you put all element selection on top and `addEventListener` in bottom.